spotirec.py

```diff
@@ -526,14 +529,14 @@ def remove_devices(devices: list):
         with open(devices_path, 'r') as file:
             saved_devices = json.loads(file.read())
     except json.decoder.JSONDecodeError:
-        print('You have no saved devices')
+        logging.warning('You have no saved devices')
```

Comment:
I think generally messages followed by an exit should be errors.

Reply:
Fixed in d3df488